tests/playToolTests.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
```

Comment:
I'm concerned that this preamble should stay in sync with the one in `playTests.sh` but may fall out of sync as `playTests.sh` will likely get more attention than this narrower-purpose/smaller script. And this setup stuff is largely to support exotic/non-standard platforms, where it's easy for us to blindly miss something.

A safer approach is probably to share this code in some shared file that can be included from both scripts (via `source` or `.`, so that it effects the caller's environment).